Tests/LogicTests/SNTXPCConnectionTest.m

```diff
@@ -16,16 +16,15 @@
 
 #import <MOLCodesignChecker/MOLCodesignChecker.h>
 #import <OCMock/OCMock.h>
+#import <MOLXPCConnection/MOLXPCConnection.h>
 
-#import "SNTXPCConnection.h"
-
-@interface SNTXPCConnectionTest : XCTestCase
+@interface MOLXPCConnectionTest : XCTestCase
```

Comment:
This exists: https://github.com/google/macops-MOLXPCConnection/blob/master/Tests/MOLXPCConnectionTests.m